### DISCUSSION TITLE
An animation app for manim?

### DISCUSSION THREAD

**Discussion by 97I8TOE47K** [3b1b/manim#1494] [Ideas]
It would be wonderful if someone will no knowledge of python or LaTeX may be able to use manim to animate math. I think if someone is into it, the community can develop an app for Windows and maybe also macOS, using which one can use manim without any knowledge of python.

**Comment by kilacoda-old** [comment]
This has been previously brought up in our [discord server](https://discord.gg/mMRrZQW) as well, but I believe we've never really had a consensus on what such an app would provide.

From what I can infer, you seem to be envisioning an app like After Effects or Blender with a drag n' drop type of interface? Or is there some other functionality you have in mind not already provided by such existing apps that this hypothetical manim app could?

**Reply by 97I8TOE47K** [reply]
All I want with the hypothetical manim app is that it should be simple to use. Blender requires a lot of knowledge of its interface and features.

**Reply by 97I8TOE47K** [reply]
More like Adobe Animate with some features of Blender, but in the context of manim

**Comment by carteeeee** [comment]
I'm sure this would be a great idea! You would have to patch some things up and make it clean to install (not just running something like `os.popen("pip3 install manim, manimlib, manimgl")`)

I would be interested in doing a collab on this though! (I do not have windows running on a computer, but I do have xCode on MacOS)

**Comment by suucss** [comment]
It is a good idea! I was anxious and worried how to install it to my computer, for it is lack of educating information.Besides, I am convinced that if an app is designed for those novices, the popularity of manim will jump into a higher stage.

**Comment by paulBrndt** [comment]
It’s been a while since anything happened here, but I still think this is a great idea! You’re absolutely right: many more people will start using Manim if it’s accessible to those who don’t code — and it could become a powerful tool in schools and education.
I’m also a Mac user, and I agree that a Mac app could be a solid starting point for an app-focused branch. Maybe we can gather a small team and collaborate on this? I’m not that experienced with Xcode yet, but I’d be happy to contribute and learn!
